plugin/tagbar.vim

```diff
@@ -146,6 +146,8 @@ command! -nargs=1 TagbarGetTypeConfig call tagbar#gettypeconfig(<f-args>)
 command! -nargs=? TagbarDebug         call tagbar#debug#start_debug(<f-args>)
 command! -nargs=0 TagbarDebugEnd      call tagbar#debug#stop_debug()
 command! -nargs=0 TagbarTogglePause   call tagbar#toggle_pause()
+command! -nargs=0 TagbarJump   call tagbar#jump()
```

Comment:
Somehow (I'm not quite sure how) this command needs to only be available in Tagbar windows. In other places it throws an error.

Reply:
Can we add the following into the `tagbar#jump()` routine to address this concern?
```
if $filetype !=# 'tagbar'
    " Not in tagbar window - ignore this function call
    return
endif
```